src/runtime_src/core/tools/common/XBMain.cpp

```diff
@@ -171,11 +172,10 @@ void  main_(int argc, char** argv,
       throw xrt_core::error(std::errc::operation_canceled);
     }
 
-    // We have a match, get first entry
-    for (auto &kd : available_devices) {
+    // We have only 1 item in the array, get it
+    for (const auto &kd : available_devices) 
       sDevice = kd.second.get<std::string>("bdf"); // Exit after the first item
-      break;
-    }
+

```

Comment:
instead of for loop can we not do:
`available_devices[0].second.get<std::string>("bdf");`?